test/integration/package_int_test.go

```diff
@@ -670,6 +670,33 @@ func (suite *PackageIntegrationTestSuite) TestRuby() {
 	cp.ExpectExitCode(0)
 }
 
+// TestProjectWithOfflineInstallerAndDocker just makes sure we can checkout and install/uninstall
+// packages for projects with offline installers and docker runtimes.
+func (suite *PackageIntegrationTestSuite) TestProjectWithOfflineInstallerAndDocker() {
+	suite.OnlyRunForTags(tagsuite.Package)
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	ts.LoginAsPersistentUser()
```

Comment:
Is this necessary? The project is public and this only slows down the test.

Reply:
Yes it is. I had a slack conversation in the buildplanner channel about this yesterday: https://activestate.slack.com/archives/C03T15HMBMG/p1695155288925919. Originally I wasn't logged in. Once I was, the problem went away.

Reply:
Ahh, I see... it makes sense since installers are an enterprise feature you would have to be logged in and a part of the org to make changes. Could you add a quick comment that explains this? I could see someone (me) looking at this test later and removing that line because it "shouldn't" be necessary